src/app/project/documents/project-document-table-rows/project-document-table-rows.component.ts

```diff
@@ -19,23 +19,50 @@ window['encodeURIComponent'] = (component: string) => {
   styleUrls: ['./project-document-table-rows.component.scss']
 })
 
-export class DocumentTableRowsComponent implements OnInit, TableComponent {
+export class DocumentTableRowsComponent implements OnInit, OnDestroy, TableComponent {
+  private ngUnsubscribe: Subject<boolean> = new Subject<boolean>();
   @Input() data: TableObject;
   @Output() selectedCount: EventEmitter<any> = new EventEmitter();
 
   public documents: any;
   public paginationData: any;
-
+  private lists: any[] = [];
   constructor(
+    private route: ActivatedRoute,
     private api: ApiService,
     private router: Router
   ) { }
 
   ngOnInit() {
+    this.route.data
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe((res: any) => {
+        if (res) {
+          if (res.documentsTableRow && res.documentsTableRow.length > 0) {
+            this.lists = res.documentsTableRow;
+          } else {
+            alert('Uh-oh, couldn\'t load list');
+            this.lists = [];
+          }
+        }
```

Comment:
When we update the lists you need to instruct angular to detect changes.

Reply:
I'm getting odd console errors when going to the documents tab on public after this patch.

Reply:
I change the order of getting paginationData and list item, console error will not pop up again. Instead of trying to finish ticket sooner, I will double check more carefully next time b4 I sent pr. Thanks